src/ArticleList.jsx

```diff
@@ -1,22 +1,28 @@
 import React, { useState, useEffect } from 'react'
-import axios from 'axios'
 import ArticleCard from './ArticleCard'
+import { getArticles } from './utils/api'
 
 function ArticleList() {
   const [articles, setArticles] = useState([])
+  const [loading, setLoading] = useState(true)
 
   useEffect(() => {
-    axios.get('https://nc-news-42oi.onrender.com/api/articles')
-        .then((response) => {
-        setArticles(response.data.articles)
+    getArticles()
+        .then((articles) => {
+        setArticles(articles);
+        })
+        .then (() => {
+          setLoading(false)
         })
         .catch((error) => {
```

Comment:
Currently when there is an error (eg network is offline), 'loading articles' still appears. Think about what you could do in this .catch block to show the user that there is an error, rather than just showing the error in the console.